safe_transaction_service/history/admin.py

```diff
@@ -328,8 +331,39 @@ class MultisigTransactionAdmin(BinarySearchAdmin):
     list_select_related = ("ethereum_tx",)
     ordering = ["-created"]
     raw_id_fields = ("ethereum_tx",)
+    readonly_fields = ("safe_tx_hash",)
     search_fields = ["=ethereum_tx__tx_hash", "=safe", "=to", "=safe_tx_hash"]
 
+    # TODO get_object and save_model is hack due to how BinaryField works
```

Comment:
Yes, I thought of changing the field but I'm not really sure if we can do that. I refuse to think Django didn't think of this. Problem is that if you don't do that you will get the field prepopulated with text "memory view at 0x1234..." and if you try to save the object again without modifying it will fail

Reply:
I think this is the only way around I found

Reply:
I found a cleaner way to do that using an `HexField`